lte/gateway/python/magma/pipelined/tests/test_ifaces.py

```diff
@@ -0,0 +1,26 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+
+from magma.pipelined.ifaces import get_mac_address
+
+
+class InterfacesTest(unittest.TestCase):
```

Comment:
IMHO deriving from unittest.Testcase is not the way to go for python tests. See: https://docs.pytest.org/en/7.1.x/how-to/unittest.html